### DISCUSSION TITLE
CoApp brokan on Linux Mint 21.3 / FF 124

### DISCUSSION THREAD

**Discussion by liar666** [aclap-dev/video-downloadhelper#408] [Help and Q&A]
### Your question

Installing coapp works perfectly and says everything is OK, but the app does not work. Videos are just "Queued" and Settings shows an blank field:
![screenshot_2024-04-18-20 16](https://github.com/aclap-dev/video-downloadhelper/assets/3216927/06c2837f-d975-4099-983c-caff63d5a7eb)

Linux Mint does not use 'Snap's (that's why I use it...) so could it be related to this bug not being fixed outside of snap?
https://github.com/aclap-dev/video-downloadhelper/discussions/12

### Addon & CoApp version and details

```
$ /opt/vdhcoapp/vdhcoapp --version
2.0.19
```
Using 
```
$ /opt/vdhcoapp/vdhcoapp install
```
also succeeds but does not solve the problem

**Comment by paulrouget** [comment]
Does restarting firefox fix the issue?

Is it flatpak Firefox or un-sandboxed firefox? (you can see in the firefox about page)

**Comment by kreuvf** [comment]
Native Linux Mint 21.3 Firefox is unsandboxed and I ran into the same problem yesterday. The solution of running `/opt/vdhcoapp/vdhcoapp install` as the user running the Firefox worked for me. I noticed that when installing the `.deb` it runs the same routine (but as root) and will create `/root/.mozilla/native-messaging-hosts/net.downloadhelper.coapp.json` and `/root/.var/app/org.mozilla.firefox/.mozilla/native-messaging-hosts/net.downloadhelper.coapp.json`.

Not sure about the intricacies of Debian package files, but I think the `install` should be issued from the correct user.
